tests/zenko_tests/configuration.py

```diff
@@ -0,0 +1,163 @@
+
+from e2e_config import accounts, endpoints, locations, workflows, schema
+
+logging.basicConfig(level=logging.INFO)
+_log = logging.getLogger("end2end configuration")
+
+
+def load_k8s_config(config_file=""):
+    """
+    Loads K8S config.
+
+    :param config_file: Name of the kube-config file.
+    """
+    try:
+        config.load_incluster_config()
+        return
+    except:
+        _log.info("unable to load incluster config")
+
+    try:
+        config.load_kube_config(config_file=config_file)
+        return
+    except:
+        _log.info("unable to load k8s config file")
+
+    raise Exception("Failed to load k8s config")
+
+
+def load_end2end_config(config_file=""):
+    """
+    Loads end-to-end test setup file
+    """
+    try:
+        with open(config_file) as file:
+            return yaml.full_load(file)
+    except:
+        raise Exception("Failed to load end2end config file")
+
+
+def create_ui_token(key, uuid):
+    """
+    Generates token for ui api
+
+    :param key
+    """
+    TTL_TIME = 60 * 60  # 1HR
+
+    if key == "":
+        raise Exception("Invalid Signing Key")
+
+    payload = {
+        "role": "user",
+        "instanceIds": [uuid],
+        "userId": "end2end",
+        "exp": (time.time()) + TTL_TIME,
+    }
+
+    return jwt.encode(payload=payload,
+                      key=key,
+                      algorithm="HS256").decode("utf-8")
+
+
+def create_swagger_client(swagger_conf_url, api_url, token=""):
+    # to view available clients
+    # print(dir(client))
+
+    # to use a client
+    # ui_client = client.ui_facing
+
+    # to list methods available
+    # print(dir(client.ui_facing))
+
+    # describe method
+    # print(client.ui_facing.createConfigurationOverlayUser.__doc__)
+
+    # to view list of models for request/response
+    # for model in client.swagger_spec.definitions:
+    #     print(model)
+
+    # retrieve model for request/response
+    # client.get_model('user-v1')
+
+    # print the properties of the model
+    # print(client.get_model('user-v1').__doc__)
+
+    if api_url.startswith("http://"):
+        api_url = api_url.lstrip("http://")
+    elif api_url.startswith("https://"):
+        api_url = api_url.lstrip("https://")
+
+    http_client = RequestsClient()
+    http_client.set_api_key(api_url,
+                            token,
+                            param_name="X-Authentication-Token",
+                            param_in="header")
+    client = SwaggerClient.from_url(swagger_conf_url, http_client=http_client)
+
+    return client
+
+
+def main():
+    KEY = os.getenv("KEY")
+    UUID = os.getenv("UUID")
+    CONFIG_FILE = os.getenv("CONFIG_FILE", "./e2e-config.yaml")
+    MANAGEMENT_ENDPOINT = os.getenv("MANAGEMENT_ENDPOINT",
+                                    "http://managementapi.zenko.local")
+    MANAGEMENT_ENDPOINT = MANAGEMENT_ENDPOINT.rstrip('/')
+    NAMESPACE = os.getenv("NAMESPACE", "default")
+
+    client = None
+    e2e_config = None
```

Comment:
I don't think there is a need to initialize these two here

Reply:
left over when using multiple try/catch blocks. will fix